config/jobs/periodic/docker/periodic-ci-docker.yaml

```diff
@@ -0,0 +1,207 @@
+periodics:
+    - name: periodic-config-docker
+      cluster: k8s-ppc64le-cluster
+      labels:
+        preset-build-docker: "true"
+      cron: 0 0 * * *
+      decorate: true
+      extra_refs:
+        - base_ref: main
+          org: ppc64le-cloud
+          repo: docker-ce-build
+      reporter_config:
+          slack:
+            channel: 'prow-job-notifications'
+            report_template: 'Job {{.Spec.Job}} of type {{.Spec.Type}} ended with state {{.Status.State}}. <!subteam^S02N6DWBX0F> <{{.Status.URL}}|View logs>'
+      spec:
+        containers:
+        - image: quay.io/powercloud/docker-ce-build@sha256:b89afc315f96f5cfd43de82b3b02f9221983592b893f838d816db5954088c9fc
```

Comment:
We typically version the images in `quay.io` and use them in Prow jobs. Using version tags here would make this line cleaner and more maintainable.

Reply:
I’m not fully convinced yet. Perhaps we could discuss this further offline?